### PR TITLE
WIP Redo the way the log file permissions are set

### PR DESCRIPTION
**Pull request by pwplusnick**
Uses a combination of setguid sticky bit and ACL to enforce log file
permissions. These changes can be found in:
roles/cinder-common/tasks/main.yml
roles/glance/tasks/main.yml
roles/keystone/tasks/main.yml
roles/neutron-common/tasks/main.yml

### CODE REVIEW

**Comment by masteinhauser** [comment]
This looks good. Only suggestion is to add a name to each task as in below:
```
- name: set cinder log dir acl permissions
  acl: name=/var/log/cinder state=present entry={{ item }}
  with_items:
    - <snip>
```

A question, do we want to add `- "default:other::r"` for the files since this may conflict with the parent directory?

**Comment by pwplusnick** [comment]
That is a good point, but I think that it is probably fine. All that is going to allow them to do is list the files in the directory.

File: roles/cinder-common/tasks/main.yml [hunk 1]
```diff
@@ -35,7 +35,13 @@
     - /var/lib/cinder/lock
 
 - name: cinder log dir
-  file: dest=/var/log/cinder state=directory mode=0755 owner=cinder group=cinder
+  file: dest=/var/log/cinder state=directory mode=2750 owner=cinder group=adm
+
+- acl: name=/var/log/cinder state=present entry={{ item }}
```
Comment (j2sol) [inline roles/cinder-common/tasks/main.yml]: This task needs a name. Every place we run into a task without a name please add a name.

Comment (j2sol) [inline roles/cinder-common/tasks/main.yml]: `entry` argument is deprecated. The preferred method is to use 
`entity`, `etype`, and `permissions`. You'll want to construct your loop differently, to loop over a hash for this data.

**Comment by j2sol** [comment]
ok to test

**Comment by j2sol** [comment]
:+1: Good to merge when tests pass.

**Comment by j2sol** [comment]
retest

**Comment by j2sol** [comment]
Looks like we need a task to get the `acl` package installed. Could put it in the `roles/common/tasks/system-tools.yml` file

**Comment by pwplusnick** [comment]
I added acl to the list that gets installed.

**Pull request merged**